src/2/world6.md

```diff
@@ -542,10 +543,10 @@ Even though I've been entrusted by Sid with this, what a mess it turned out to b
 "What do you mean! By Sid?"
 
 It was the same as with Dark Empress Vanessa.
-Just before change due to Zero Code leads to their death, other races heroes remembered about Sid.
-It became somewhat common point now.
+Just before the change due to Zero Code leads to their death, other race's heroes remembered about Sid.
```

Comment:
I see slight mistake on my part here

`heroes` -> `hero`

`Just before the change due to Zero Code leads to their death,`

Should be: `Just before they disappear due to Zero Code change`

Reply:
This might be a roundabout way of saying this, but how do you think of the following:

> It was the same as with Dark Empress Vanessa.
> Just before disappearing due to the Zero Code change, the other race's hero had remembered about Sid.
> It was a common point between them.